.travis.yml

```diff
@@ -0,0 +1,49 @@
+language: c
+
+dist: xenial
+
+matrix:
+  include:
+    - env: GHCVER=8.6.3 CABALVER=2.4.1.0
+      os: linux
+      sudo: required
+    - env: GHCVER=8.4.4 CABALVER=2.4.1.0
+      os: linux
+      sudo: required
+
+addons:
+  apt:
+    packages:
+      - libnuma-dev
+
+before_install:
+  - .ci/travis-install.sh
+  - export PATH="$HOME/.cabal/bin:$HOME/.ghcup/bin:$PATH"
+
+install:
+  - cabal v2-update
+  - cabal v2-install -j --overwrite-policy=always stylish-haskell hlint
+
+script:
+  - .ci/check-haskell-format.sh
+  - hlint .
+  - cabal v2-configure --enable-tests
+  # wtflol:
+  - 'echo -e "package gossip\n  ghc-options: -Werror" >> cabal.project.local'
+  - cat cabal.project.local
```

Comment:
Do we still need this?

Reply:
Yes. I'm at a point where I don't trust the cabal folks anymore to not silently introduce some breakage to `configure`, so I'd rather want to know what the configuration is.